src/pages/docs/dynamic-servers/concepts/monitoring.mdx

```diff
@@ -0,0 +1,14 @@
+import imgMonitoring from './images/monitoring.png';
+
+# Monitoring
+
+Rivet provides a system for monitoring hardware metrics & lobby crashes in real-time for free. This is a great way to monitor your game's performance and stability. Your server is monitored automatically and can be viewed on the Rivet Hub.
+
+<Image src={imgMonitoring} alt='Monitoring dashboard' />
+
+## Memory usage
```

Comment:
Is there a PR for changing the link in `src/elements/pages/dev/game/pages/game-logs.ts` to point here instead of the deleted doc?

Reply:
https://github.com/rivet-gg/hub/pull/114